### DISCUSSION TITLE
500 Server Error after an update

### DISCUSSION THREAD

**Discussion by kmanan** [linuxserver/Heimdall#1410] [Q&A]
Hi. Running Heimdall in Portainer in Ubuntu on Proxmox. Everything was working fine, I installed watchtower yesterday, and it updated the images. Heimdall has stopped working. I keep getting the 500 Server Error. I looked at the logs and am seeing this. I don't know / understand the space error or what to do about it.

> 166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>    ErrorException 
>   fwrite(): Write of 19357 bytes failed with errno=28 No space left on device
>   at /app/www/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:162
>     158▕      * @param resource $stream
>     159▕      */
>     160▕     protected function streamWrite($stream, LogRecord $record): void
>     161▕     {
>   ➜ 162▕         fwrite($stream, (string) $record->formatted);
>     163▕     }
>     164▕ 
>     165▕     private function customErrorHandler(int $code, string $msg): bool
>     166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>    ErrorException 
>   fwrite(): Write of 19357 bytes failed with errno=28 No space left on device
>   at /app/www/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:162
>     158▕      * @param resource $stream
>     159▕      */
>     160▕     protected function streamWrite($stream, LogRecord $record): void
>     161▕     {
>   ➜ 162▕         fwrite($stream, (string) $record->formatted);
>     163▕     }
>     164▕ 
>     165▕     private function customErrorHandler(int $code, string $msg): bool
>     166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>    ErrorException 
>   fwrite(): Write of 19357 bytes failed with errno=28 No space left on device
>   at /app/www/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:162
>     158▕      * @param resource $stream
>     159▕      */
>     160▕     protected function streamWrite($stream, LogRecord $record): void
>     161▕     {
>   ➜ 162▕         fwrite($stream, (string) $record->formatted);
>     163▕     }
>     164▕ 
>     165▕     private function customErrorHandler(int $code, string $msg): bool
>     166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>    ErrorException 
>   fwrite(): Write of 19357 bytes failed with errno=28 No space left on device
>   at /app/www/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:162
>     158▕      * @param resource $stream
>     159▕      */
>     160▕     protected function streamWrite($stream, LogRecord $record): void
>     161▕     {
>   ➜ 162▕         fwrite($stream, (string) $record->formatted);
>     163▕     }
>     164▕ 
>     165▕     private function customErrorHandler(int $code, string $msg): bool
>     166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>    ErrorException 
>   fwrite(): Write of 19357 bytes failed with errno=28 No space left on device
>   at /app/www/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:162
>     158▕      * @param resource $stream
>     159▕      */
>     160▕     protected function streamWrite($stream, LogRecord $record): void
>     161▕     {
>   ➜ 162▕         fwrite($stream, (string) $record->formatted);
>     163▕     }
>     164▕ 
>     165▕     private function customErrorHandler(int $code, string $msg): bool
>     166▕     {
>       +14 vendor frames 
>   15  /app/www/artisan:35
>       Illuminate\Foundation\Console\Kernel::handle()
>

**Comment by kmanan** [comment]
Nvm, it was in fact the storage on the VM ran out. Portainer sucked up 100GB. Fixed it by increasing the storage and running the 4-5 commands for Ubuntu to expand it.

**Comment by voodoopt** [comment]
Size on disk (Size24G  Used19G Avail4.0G Use83% )
same problem on version: v2.6.3-ls309
